resources/views/inc/trainer_feedback.blade.php

```diff
@@ -2,18 +2,18 @@
     {{ html()->form()->route('saveNewTrainerFeedback')->open() }}
         @php ($redirect_to = (isset($redirect)) ? $redirect : 'external')
         {{ html()->text('redirect_to', $redirect_to)->attribute('hidden') }}
-        <div class="row">
+        <div class="row mb-3">
```

Comment:
See above

Reply:
`<div class="row` appears in the codebase 229 times and is too generic of a selector to apply a global style like this. I am going to leave this one alone.

Reply:
Done.